ew_agents/mongodb_storage.py

```
@@ -0,0 +1,492 @@
+                connection_status = f"ping_failed: {str(ping_error)}"
+            
+            # Test database access
+            try:
+                collections = self.db.list_collection_names()
+                db_access = "accessible"
+            except Exception as db_error:
+                db_access = f"access_failed: {str(db_error)}"
+            
+            # Test collection operations
+            try:
+                test_count = await self._mcp_count_documents(self.analysis_collection, {})
```

Ruff (0.12.2)

Local variable `test_count` is assigned to but never used

Remove assignment to unused variable `test_count`

(F841)